### ISSUE TITLE
Put messages next to buttons on taxonomy and resource panels

### ISSUE THREAD

**Issue by noisecapella** [mitodl/lore#798]
I think this is what we agreed on, feel free to comment if I misheard. Was a part of #439

**Comment by jamiefolsom** [comment]
Yes, the other option we discussed was anchoring the message to the bottom of the fixed title bar at top of the panel. I like the idea of putting the message next to the button myself.
